api/src/main/java/jakarta/data/repository/Query.java

```diff
@@ -94,30 +135,30 @@
 public @interface Query {
 
     /**
-     * <p>Defines the query to be executed when the annotated method is called.</p>
+     * <p>Specifies the query executed by the annotated repository method,
+     * in JDQL or JPQL.</p>
      *
-     * <p>If an application defines a repository method with {@code @Query}
-     * and supplies other forms of sorting (such as {@link Sort}) to that method,
-     * then it is the responsibility of the application to compose the query in
-     * such a way that an {@code ORDER BY} clause (or query language equivalent)
-     * can be validly appended. The Jakarta Data provider is not expected to
-     * parse query language that is provided by the application.</p>
+     * <p>If the annotated repository method accepts other forms of sorting
+     * (such as a parameter of type {@link Sort}), it is the responsibility
+     * of the application programmer to compose the query so that an
+     * {@code ORDER BY} clause can be validly appended to the text of the
+     * query.</p>
```

Comment:
This part was lost in the rewrite:
```suggestion
     * query. The Jakarta Data provider is not expected to parse query language
     * that is provided by the application.</p>
```

Reply:
I intentionally removed it. Because a Jakarta Data provider _does_ parse JDQL. Now, perhaps it passes it on to a JPQL implementation to parse, but in that case the JPQL implementation is part of the Jakarta Data provider.